### DISCUSSION TITLE
Print monthly reports for `klog report`

### DISCUSSION THREAD

**Discussion by ghost** [jotaen/klog#308] [Help / Question] [answered]
Hi,

I would like to be able to run:
```
klog report --monthly --diff
```

This would help me to manage my tracked overtime.
Right now I am this for every month:
```
klog total --period=2024-06 --diff
```

An overview would be awesome.

**Comment by jotaen** [comment, marked as answer]
The default behaviour is for `klog report` to aggregate/summarise by days, but you can control this via the `--aggregate` option, e.g. `klog report --aggregate month` (or short `klog report -a m`). You can still combine this with `--diff`. Would this be helpful for your use-case?

By the way, instead of `klog total --period=2024-06` you can also do `klog total --last-month` or `klog total --this-month`, for convenience.

**Reply by ghost** [reply]
`--aggregate` is perfect. If I know what I am looking for, I am able to find it in documentation.

Thank you very much!
